docs/markdown/Reference-manual.md

```diff
@@ -1964,6 +1976,9 @@ the following methods.
   can be specified. If `true` the script will not be run if DESTDIR is set during
   installation. This is useful in the case the script updates system wide
   cache that is only needed when copying files into final destination.
+  *(since 0.60.0)* `install_tag` string keyword argument can be specified.
```

Comment:
Could make sense to mention the default tag value in these types of comments

Reply:
The next line is:
> By default the script has no install tag which means it is not being run when `meson install --tags` argument is specified.

I think I specify the default everywhere, no?